composer.json

```diff
@@ -6,13 +6,16 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
+        "php": "^7.1",
         "drupal/core": "^8.7",
         "drupal/language_selection_page": "^2.2",
         "drupal/pathauto": "^1.2",
-        "php": "^7.1"
+        "drupal/redirect": "^1.4"
```

Comment:
Hm...the Redirect module is not required by oe_multilingual. Only by the submodule. So we need to require it as a dev dependency and add this information to the readme. See the poetry submodule wich requires a library as an example: https://github.com/openeuropa/oe_translation/tree/POETRY/modules/oe_translation_poetry